test/libsolidity/semanticTests/array/copying/copy_dynamically_sized_storage_array.sol

```diff
@@ -0,0 +1,21 @@
+contract C {
+    function testFunction1() public {}
+    function testFunction2() public {}
+    function testFunction3() public {}
+
+
+    function() external [] externalArray0 = new function() external[] (3);
+    function() external [] externalArray1 = [this.testFunction1, this.testFunction2, this
+    .testFunction3];
+
+    function copyExternalStorageArrayOfFunctionType() external returns (bool)
+    {
```

Comment:
For some reason, we only use this style in C++ :) Not really sure why. In Solidity code this should look like this:
```suggestion
    function copyExternalStorageArrayOfFunctionType() external returns (bool) {
```